db/migrate/20160304205331_add_auth_failed_to_sms_messages.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthFailedToSmsMessages < ActiveRecord::Migration
+  def change
+    add_column :sms_messages, :auth_failed, :boolean
```

Comment:
Can we have default: false and null: false please? I like to avoid nulls in boolean columns unless they have an explicit meaning.